.github/workflows/ci.yml

```diff
@@ -17,7 +17,7 @@ jobs:
 
       - uses: actions/setup-node@v2
         with:
-          node-version: '10.17.0'
+          node-version: '10.18.0'
```

Comment:
10 is eol should we drag this up to 12.14.0 to match quilt?

Reply:
yeah... I was toying with doing there here or in a new PR.